src/pages/Calculator.tsx

```diff
@@ -174,26 +174,20 @@ export const Calculator = (): React.ReactElement => {
     </button>
   )
 
+  const oneWeekOfMilliseconds = 7 * 24 * 60 * 60 * 1000
+  const currentTimeMillis = new Date().getTime()
+  const prevalenceDataDate = calculatorData.prevalanceDataDate
+  const prevalenceDataTimeMillis = prevalenceDataDate?.getTime()
+  const prevalenceIsStale =
+    prevalenceDataTimeMillis !== null &&
+    currentTimeMillis - prevalenceDataTimeMillis > oneWeekOfMilliseconds
+
+  const showStaleWarning = prevalenceIsStale && !suppressStaleWarning
+
   return (
     <div id="calculator">
       <Row>
         <Col md="12" lg="8" id="calculator-introduction">
-          {showWarning && (
-            <Alert
-              variant="primary"
-              onClose={() => setShowWarning(false)}
-              dismissible
-            >
-              <Alert.Heading>
-                {t('calculator.intro.omicron_warning_heading')}
-              </Alert.Heading>
-              <Trans i18nKey="calculator.intro.omicron_warning">
-                Research is in progress to determine Omicron's impact on our
-                models. Preliminary findings suggest an increase in the risk for
-                re-infection.
-              </Trans>
-            </Alert>
-          )}{' '}
```

Comment:
I wasn't sure what the `{' '}` was accomplishing here and did not bring it over.  Let me know if that was a mistake.